app/src/main/java/com/codesoom/assignment/controllers/SessionController.java

```diff
@@ -0,0 +1,40 @@
+// REST API 작성하기
+// 1. CREATE -> 로그인 -> Token 발행
+// 2. READ (w/ Token) -> 세션 정보 전달 -> 세션 정보가 유효한가?, 유저 정보
+// 3. UPDATE (w/ Token) -> Token 재발행
+// 4. DELETE -> 로그아웃 -> Token 무효화
+
+// session은 여러개가 존재할 수 없으므로, 단수형을 사용한다.
+
+
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.application.AuthenticationService;
+import com.codesoom.assignment.dto.SessionResponseData;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/session")
+public class SessionController {
+    private AuthenticationService authenticationService;
```

Comment:
필드 멤버에는 `final`을 애용해 줍시다.
```suggestion
    private final AuthenticationService authenticationService;
```